video/qt_decoder.h

```diff
@@ -418,10 +429,17 @@ class QuickTimeDecoder : public VideoDecoder, public Audio::QuickTimeAudioDecode
 		Graphics::Surface *_projectedPano;
 		Graphics::Surface *_planarProjection;
 
+		// Defining these for the sake of making the swing transition happen
+		// This will be the start point of the transition
+		float _currentFOV = 0;
```

Comment:
These seem to be an artifact from the previous approach of external looping. I think you can kill them and use local variables.